scripts/test.sh

```diff
@@ -6,4 +6,4 @@ service mysql start
 go generate ./...
 go get -v -t ./...
 go install ./...
-go test -v ./...
+SQLFLOW_log_dir= go test -v ./...
```

Comment:
I'm not sure what the effect is.

Reply:
Shall we also add `SQLFLOW_log_level=debug`?

Reply:
Thanks @weiguoz @tonyyang-svail , Done.